src/tests/test_utils/unit_test_utils/mocks/cpp_interfaces/impl/mock_async_infer_request_default.hpp

```diff
@@ -11,6 +11,7 @@
 
 #include "cpp_interfaces/impl/ie_infer_async_request_thread_safe_default.hpp"
 
+using namespace ov::threading;
```

Comment:
Because `ITaskExecutor::Ptr` needs this namespace in this function `MockAsyncInferRequestDefault(IInferRequestInternal::Ptr request, const ITaskExecutor::Ptr& taskExecutor, const ITaskExecutor::Ptr& callbackExecutor)`.